scala-package/pom.xml

```diff
@@ -112,6 +112,44 @@
         </plugins>
       </build>
     </profile>
+
+    <profile>
+      <id>scala-2.11</id>
```

Comment:
do we need to move `<scala.verion>2.11.8</scala.version>` here, and add enforcer-plugin as well?